src/pynwb/ndx_hierarchical_behavioral_data/hierarchical_behavioral_data.py

```diff
@@ -56,3 +51,6 @@ def add_interval(self, **kwargs):
             kwargs.update(stop_time=self['next_tier'].target.table['stop_time'][kwargs['next_tier'][-1]])
 
         super().add_interval(**kwargs)
+
+    def to_hierarchical_dataframe(self):
+        return to_hierarchical_dataframe(dynamic_table=self)
```

Comment:
or how about keeping `to_hierarchical_dataframe`?

Reply:
ok, but there's also `to_denormalized_dataframe`

Reply:
that should be behave the same as `.to_hierarchical_dataframe().reset_index()` (we can discuss this in our meeting, I can show you)

Reply:
@bendichter I updated the readme to be able to reproduce the tables that are shown there. Let me know what you think about it. Otherwise the tests are done as well.